statuscake.tf

```diff
@@ -0,0 +1,30 @@
+provider "statuscake" {
+  api_token = local.statuscake_api_token
+
+}
+
+resource "statuscake_ssl_check" "main" {
+  count          = local.enable_monitoring && local.monitor_tls_expiry ? 1 : 0
+  check_interval = 900 # Interval in seconds to test. (15 mins)
+
+  alert_config {
+    alert_at = [7, 14, 30]
+
+    on_reminder = true
+    on_expiry   = true
+    on_broken   = false
+    on_mixed    = false
+  }
+
+  monitored_resource {
+    address = "https://www.dxw.com"
+  }
+}
```

Comment:
You've defined `statuscake_contact_group` resource further down but it needs to be linked to this ssl check resource too:
```hcl
contact_groups = [statuscake_contact_group.monitoring_team.id]
```